rita/src/main.rs

```diff
@@ -77,37 +77,42 @@ fn main() {
 
     trace!("Starting with Identity: {:?}", my_ident);
 
-    let (tx, rx) = mpsc::channel();
 
-    let tx1 = mpsc::Sender::clone(&tx);
+    let (debt_keeper_input, debt_keeper_output) = DebtKeeper::start(Int256::from(1), Int256::from(-100000000000000i64), Int256::from(100), 2u32);
```

Comment:
What makes `-100000000000000i64` special? Why not just use `Int256`'s minimum value by default?

Reply:
That is 100k micro ethers which might be around 10 cents or something. Using Int256's minimum value would result in a debt keeper which does not require counterparts to pay at all.

Reply:
Okay. IMHO we should definitely use an exponent or Rust's underscore notation here, e.g. `-1e14 as i64` or  `-100_000_000_000_000`

Reply:
This is actually a mistake, it should be something closer in magnitude to 1% the amount of wei sent in the test. I will be changing this to 1 billion (the tests usually require payments ~50Gwei).